### PR TITLE
[ICDS Dashboard]Migration Form UCR

### PR DESCRIPTION
**Pull request by ShubhamBansal1997**
##### SUMMARY

This PR is made to add a new UCR for the Migration Form, this is the sub task of this JIRA ticket https://dimagi-dev.atlassian.net/browse/ICDS-1025

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -1881,6 +1881,7 @@
     os.path.join('custom', 'icds_reports', 'ucr', 'data_sources', 'dashboard', 'daily_feeding_forms.json'),
     os.path.join('custom', 'icds_reports', 'ucr', 'data_sources', 'primary_private_school_form_ucr.json'),
     os.path.join('custom', 'icds_reports', 'ucr', 'data_sources', 'cbe_form.json'),
+    os.path.join('custom', 'icds-reports', 'ucr', 'data_sources', 'migrations_form.json'),
```
Comment (Rohit25negi) [inline settings.py]: hey, I think problem is here. Its not `icds-reports`, Its `icds_reports`

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
I think problem is here.

**Comment by czue** [comment]
I don't think I have the right context to review this so will defer to @Rohit25negi

File: custom/icds_reports/ucr/data_sources/migrations_form.json [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+          },
+          "location_type": "state",
+          "location_property": "_id"
+        },
+        "is_primary_key": false,
+        "is_nullable": true,
+        "column_id": "state_id",
+        "type": "expression"
+      },
+			{
```
Comment (Rohit25negi) [inline custom/icds_reports/ucr/data_sources/migrations_form.json]: nitpick: can you fix the formating

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Just formating nits! Else looks good. 
We might also need to consider the migrated_in case.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
can you also make this UCR distributed. If you want to see the example. you can refer https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/ucr/data_sources/adolescent_girl_register_form_ucr.json#L188

File: custom/icds_reports/ucr/data_sources/migrations_form.json [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+            "type": "named",
+            "name": "user_location_id"
+          },
+          "location_type": "supervisor",
+          "location_property": "_id"
+        },
+        "is_primary_key": true,
+        "is_nullable": false,
+        "column_id": "supervisor_id",
+        "create_index": true,
```
Comment (Rohit25negi) [inline custom/icds_reports/ucr/data_sources/migrations_form.json]: I think this property is not required here as its creating  a primary key on supervisor_id and timeend.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]

**Review by Rohit25negi — APPROVED** [review]

**Pull request merged**